pkg/storer/internal/reserve/reserve.go

```diff
@@ -70,6 +71,19 @@ func New(
 	}
 	rs.radius.Store(uint32(rItem.Radius))
 
+	epochItem := &EpochItem{}
+	err = store.Get(epochItem)
+	if err != nil {
+		if errors.Is(err, storage.ErrNotFound) {
+			err := store.Put(&EpochItem{Timestamp: uint64(time.Now().Unix())})
+			if err != nil {
+				return nil, err
+			}
+		} else {
```

Comment:
This `else` branch is not needed, replace it with `return nil, err`.

Reply:
it's needed because there is more code that needs to run below